etc/kayobe/environments/ci-multinode/stackhpc-ci.yml

```diff
@@ -51,8 +51,8 @@ stackhpc_docker_registry: "{{ stackhpc_repo_mirror_url | regex_replace('^https?:
 stackhpc_docker_registry_username: "stackhpc-kayobe-ci"
 stackhpc_docker_registry_password: !vault |
           $ANSIBLE_VAULT;1.1;AES256
-          30333932623139383562343036366566363230656536376164633463366235623463323937303135
-          6533333638613764383136613730346436643831646530640a363262373964323461616638636135
-          65623338333236323030346564363035313462333335333032643661363834356432363833303838
-          3936616462303062350a316165396265313766316338303338336132316264316631376163643133
-          33616239633435636630373039616630323163343962336330303864626561303238
+          33356166343730633865363431306535613736663764373034396132356131343066636530393534
+          3262646436663034633131316438633230383330633533350a386365313239303464383636376338
+          61656662333939333063343131633963636431663136643137636664633233633133396339613861
+          3038613063626138610a333566393937643630366564653163613364323965396130613433316537
+          39653335393831633362343934363866346262613166393561666336623062393935
```

Comment:
Has this been rekeyed? If so who has the password?

Reply:
@grzegorzkoper